CHANGELOG.md

```diff
@@ -1,99 +1,105 @@
+# 2.1.1
+
+> Jun 04, 2021
+
+- :nut_and_bolt: **New** Added Switch.
+
 # 2.1.0
 
 > Apr 12, 2021
 
-* :tada: **Enhancement** Reintroduce the fhi-app and fhi-main-menu blocks into the package
```

Comment:
Are all this changes (hole file, not just this line) something you did deliberately, or is it something your editor did?
If deliberately, ok by me since I don't have strong opinions ether way, but then we should do it in all Frontend repos maybe... for consistensy.